Stack/Opc.Ua.Core/Security/Certificates/ICertificateStore.cs

```diff
@@ -86,8 +86,20 @@ public interface ICertificateStore : IDisposable
         /// <param name="password">The certificate password.</param>
         /// <remarks>Returns always null if SupportsLoadPrivateKey returns false.</remarks>
         /// <returns>The matching certificate with private key</returns>
+        [Obsolete("Must specify certificate type.")]
```

Comment:
This API should default to RSA, and the comment should mention it for backward compatibility.